From b8071e54e181e6e5e11bcc6d33e1e83047744e4a Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Sun, 21 Mar 2021 11:44:13 +0200
Subject: [PATCH 1356/1921] dts: marvell: Enable 10G interfaces on 9131-DB
 boards

This patch enables eth3 10G interface on CN9131-DB.

Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Change-Id: I3f292ec9ba4a61f58760be138a23b268b02376fa
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/48232
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9131-db.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9131-db.dtsi b/arch/arm64/boot/dts/marvell/cn9131-db.dtsi
index b3846bedf216..8e3f1381cb21 100644
--- a/arch/arm64/boot/dts/marvell/cn9131-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9131-db.dtsi
@@ -86,7 +86,7 @@
 
 /* CON50 */
 &cp1_eth0 {
-	status = "disabled";
+	status = "okay";
 	phy-mode = "10gbase-kr";
 	/* Generic PHY, providing serdes lanes */
 	phys = <&cp1_comphy4 0>;
-- 
2.31.1

